### DISCUSSION TITLE
Class "Spatie\Analytics\Facades\Analytics" not found

### DISCUSSION THREAD

**Discussion by kjoneswatts** [freekmurze/laravel-analytics#519] [Q&A]
I've got this issue that I have not been able to figure out. When in my localhost, everything works wonderfully, no errors, but the second I push to live an error is thrown.

Class "Spatie\Analytics\Facades\Analytics" not found 

I've followed the ReadMe file and set up everything exactly the same, it works fine on my local development, but not live.

use Spatie\Analytics\Facades\Analytics;
use Spatie\Analytics\Period;

use Google\Analytics\Data\V1beta\Filter;
use Google\Analytics\Data\V1beta\FilterExpression;
use Google\Analytics\Data\V1beta\Filter\StringFilter;
use Google\Analytics\Data\V1beta\Filter\inListFilter;
use Google\Analytics\Data\V1beta\Filter\StringFilter\MatchType;

$theOrganizationAnalytics = Analytics::get(Period::days($theAnalyticsRange), ['screenPageViews', 'screenPageViewsPerUser', 'totalUsers', 'userEngagementDuration'], [], 0, [], 0, $theOrganizationAnalyticsDimensionFilter);

Please help. I'm at a loss.

**Comment by MarvinVk** [comment]
Just using `use Analytics;` fixed this issue for me
